pkg/sdk/validation/manifest/schema_validate.go

```diff
@@ -10,9 +10,9 @@ import (
 // jsonSchemaCollection defines JSONSchema collection index by the name.
 type jsonSchemaCollection map[string]string
 
-// validateJSONSchema07Definition validate a given JSONSchema collection.
+// checkJSONSchema07Definition validate a given JSONSchema collection.
```

Comment:
```suggestion
// checkJSONSchema07Definition validates a given JSONSchema collection.
```

But TBH I liked `validateJSONSchema07Definition` more 🙂

Reply:
me to, but I did it for consistency which in this case IMO is more important